lib/event_framework/bounded_context.rb

```diff
@@ -80,10 +80,15 @@ def build_command_dependency_chain!(event_store_database: :event_store)
         end
 
         container.register("event_store.sink") do
-          EventStore::Sink.new(
+          tracer = container.key?("tracer") ? container["tracer"] : nil
```

Comment:
Does `container["tracer"]` not return `nil` if it doesn't have a tracer?

Reply:
Dry::Container actually raises an exception if you try and resolve a non-existent entry, which is why I need this check here.